src/routes/examples/react/using-factories.mdx

```diff
@@ -0,0 +1,221 @@
+# Using Factories
+
+This example builds on the _Resource Helper_ example with the
+following enhancements:
+
+* MirageJs is configured to use a factory instead of a fixture
+* Faker.js is introduced as a development dependency
+* Tests will use MirageJs's `server` to generate `Country`
+records as required
+
+## Faker.js
+
+`faker` is optional.  This configuration example uses it in
+conjunction with MirageJs' factory mechanism.  To learn more visit
+[faker.js](https://github.com/Marak/faker.js).  Installation is
+trivial:
+
+```bash
+yarn add --dev faker
+```
+
+## Configuring A Factory
+
+Start by creating a basic configuration that can be invoked from the setup and
+teardown hooks in Jest.
+
+```js
+// src/mirage.js
+
+import { Factory, Model, Server } from '@miragejs/server';
+import Faker from 'faker';
+
+let startMirage = function({ environment = 'test' } = {}) {
+  return new Server({
+    environment,
+
+    // -------------------------------- 🎉🎉🎉 New Section 🎉🎉🎉
+    // configure Mirage to be aware that it
+    // can be used to generate `Country` records
+    factories: {
+      country: Factory.extend({
+        name: () => Faker.address.country() // using Faker.js
+      })
+    },
+    // ------------------------------------ 🎉🎉🎉 End New 🎉🎉🎉
+
+    // declare the models that Mirage needs
+    // to know how to serve
+    models: {
+      country: Model
+    },
+
+    baseConfig() {
+      // routes below `this.urlPrefix` assignment are
+      // expected to resolve to `https://some.example.com`
+      this.urlPrefix = 'https://some.example.com';
+
+      // EXAMPLE: the resource shorthand will expose the
+      // GET/POST/DELETE/PATCH endpoints;
+      // e.g. GET/POST of https://some.example.com/countries
+      this.resource('countries');
+
+      // all other API requests are passed
+      // through to their destinations
+      this.passthrough();
+    }
+  });
+};
+
+export { startMirage };
+```
+
+## CRUD Class
```

Comment:
Same _CRUD_ class from the resource helper example.  Should I put this in a `snippet`?  I haven't use Gatsby until now, so I am not sure what facilities I have at my disposal.